### DISCUSSION TITLE
Documentation on Management System module and its related sub-modules

### DISCUSSION THREAD

**Discussion by 007ke** [OCA/odoo-community.org#96] [Q&A]
I was hoping if someone here can help or guide me in gaining any kind of detailed documentation / user guide for the OCA module https://github.com/OCA/management-system and all its sub-modules so that it gives a good understanding on how to best install and use it. 
Thank you in anticipation

**Comment by etobella** [comment]
You can try with the following video explaining how to manage it :smile: 
https://www.youtube.com/watch?v=w2nf_O9TajM
